### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by hayleyw7**
# Thank you for pushing your code!

## What was implemented?
- Start README

## Were there any issues that arose?
-

## Is there anything that you need from your teammate?
- We still need your reflections
- We still need a gif
- We still need to make all of the subheadings under Reflections the same level

## Any other comments, questions, or concerns?
-

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [ ] open index.html
- [ ] console.log()
- [ ] dev tools

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,86 +2,69 @@
...
 
-Built with Visual Studio Code
-
-[Code stored on GitHub](https://github.com/williamphelps13/rancid-tomatillos)
-
-[Project board on GitHub](https://github.com/williamphelps13/rancid-tomatillos/projects/1) 
+## Middle of project
 
-<!-- 
+A challenge was setting up Router.
```
Comment (williamphelps13) [inline README.md:29]: We could add more detail here

File: README.md [hunk 2]
```diff
@@ -2,86 +2,69 @@
...
-Enter the following command in the Terminal:
-`npm install`
+## Languages and technology
 
-### Run local host
+Written in JavaScript, HTML, CSS, & JSX
 
-Enter the following command in the Terminal:
-`npm start`
+Utilizes NPM, Node, Cypress, Router, & React
```
Comment (williamphelps13) [inline README.md:43]: Can you add Surge?

File: README.md [hunk 3]
```diff
@@ -2,86 +2,69 @@
...
 
-Utilizes NPM, Node, Cypress, Router, & React
+Big successes were setting up React components, application foundations, & implementing JSX.
 
-Built with Visual Studio Code
-
-[Code stored on GitHub](https://github.com/williamphelps13/rancid-tomatillos)
-
-[Project board on GitHub](https://github.com/williamphelps13/rancid-tomatillos/projects/1) 
+## Middle of project
```
Comment (williamphelps13) [inline README.md:27]: Cn you make this and start of project a ###?

**Review by williamphelps13 — CHANGES_REQUESTED** [review]
There were just few changes needed added in comments!

File: README.md (hunk 2 above)
Comment (hayleyw7) [reply README.md:45]: Done!

Comment (williamphelps13) [reply README.md:45]: Nice!

**Review by williamphelps13 — APPROVED** [review]

**Pull request merged**